Backend/DynamicQR/DynamicQR.Infrastructure/Services/QrCodeTargetRepositoryService.cs

```diff
@@ -22,7 +23,10 @@ public async Task<bool> CreateAsync(QrCodeTarget qrCodeTarget, CancellationToken
 
         Azure.Response response = await _tableClient.AddEntityAsync(qrCodeTargetData, cancellationToken);
 
-        return !response.IsError;
+        if (response.IsError)
+            throw new StorageException(response.ReasonPhrase);
+
+        return true;
```

Comment:
@jensdenbraber hier kan deze ook weg. Op andere plekken in de code ook waar de StorageException aan toe is gevoegd.